<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{color: blue; font: bold 10pt;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="plainText">
       <string>Введите текст</string>
      </property>
      <property name="placeholderText">
       <string>Введите текст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">QPushButton:hover{color: blue; font: bold 10pt;}</string>
      </property>
      <property name="text">
       <string>Сохранить картинку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Загрузить картинку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Декодировать текст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Закодировать текст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">&lt;span style=&quot;font-size:10pt; color:#ff0000;&quot;&gt;&lt;b&gt;Привет:&lt;/b&gt;&lt;/span&gt;</string>
      </property>
      <property name="text">
       <string>&lt;span style=&quot;font-size:10pt; color:#ff0000;&quot;&gt;&lt;b&gt;Привет:&lt;/b&gt;&lt;/span&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
